src/components/NavigationAccount.vue

```diff
@@ -21,12 +21,11 @@
   -->
 
 <template>
-	<AppNavigationItem v-if="visible"
+	<NcAppNavigationCaption v-if="visible"
 		:id="id"
 		:key="id"
 		:menu-open.sync="menuOpen"
```

Comment:
Does this prop exist? https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppNavigation?id=ncappnavigationcaption